omx/gstomxvideodec.c

```diff
@@ -2354,18 +2351,13 @@ gst_omx_video_dec_handle_frame (GstVideoDecoder * decoder,
   guint offset = 0, size;
   GstClockTime timestamp, duration;
   OMX_ERRORTYPE err;
-  gboolean flush_flag;
 
   self = GST_OMX_VIDEO_DEC (decoder);
   klass = GST_OMX_VIDEO_DEC_GET_CLASS (self);
 
   GST_DEBUG_OBJECT (self, "Handling frame");
 
-  GST_VIDEO_DECODER_STREAM_LOCK (self);
-  flush_flag = self->flush_flag;
-  GST_VIDEO_DECODER_STREAM_UNLOCK (self);
-
-  if (flush_flag) {
+  if (self->flush_flag) {
```

Comment:
keep protection

Reply:
Added in 649bdbaf446bccb9760859a2c0f8ea3e5ad6f03c